cmd/heartbeats/main.go

```diff
@@ -94,12 +102,14 @@ func main() {
 		ceOverrides = &overrides
 	}
 
-	p, err := cloudevents.NewHTTP(cloudevents.WithTarget(sink))
+	conf, err := config.JSONToTracingConfig(env.TracingConfig)
 	if err != nil {
-		log.Fatalf("failed to create http protocol: %s", err.Error())
+		log.Printf("Failed to read tracing config, using the no-op default: %v", err)
 	}
-
-	c, err := cloudevents.NewClient(p, cloudevents.WithUUIDs(), cloudevents.WithTimeNow())
+	if err := tracing.SetupStaticPublishing(zap.L().Sugar(), "", conf); err != nil {
```

Comment:
should this be inside the else from the err on `JSONToTracingConfig` ? seems like if the like 105 err was not nil, then conf will be nil?

Reply:
The conf will be a NoopConfig, which means this will basically noop as well.